stdlib/public/runtime/ImageInspectionELF.cpp

```diff
@@ -72,6 +72,13 @@ static SectionInfo getSectionInfo(const char *imageName,
   SectionInfo sectionInfo = { 0, nullptr };
   void *handle = dlopen(imageName, RTLD_LAZY | RTLD_NOLOAD);
   if (!handle) {
+#ifdef __FreeBSD__
```

Comment:
I think this bit is fine although it duplicates the logic. Can you try to share the code between FreeBSD and android?